### DISCUSSION TITLE
inspector label

### DISCUSSION THREAD

**Discussion by Lijesh-km** [clientIO/joint#1977] [Q&A] [answered]
[ui.Inspector.custom-fields](https://resources.jointjs.com/docs/rappid/v3.6/ui.Inspector.html#ui.Inspector.custom-fields)
![label](https://user-images.githubusercontent.com/121955632/211165459-ea120876-9e5b-49f4-a242-d14ea7a31da0.png)
With this resource, I create a new list in the inspector.  But I need a path label instead of the text label, how can I do that?

**Comment by kumilingus** [comment, marked as answer]
Unless you use explicitely set `label` property, the path is the default value for the label.

```js
  test1: {
      type: 'list',
      item: {
          type: 'object',
          properties: {
              test2: {
                  type: 'text',
                  // Note: uncomment to change label from `test1/0/test2` to `test2 label`
                  // label: 'test2 label'
              }
          }
      }
  }
```

If you need to customize labels see [Inspector Custom Labels](https://resources.jointjs.com/docs/rappid/v3.6/ui.Inspector.html#ui.Inspector.custom-labels) tutorial.
